contactsPane.js

```diff
@@ -1180,21 +1183,49 @@ module.exports = {
             }
           }
 
+          function trashCan () {
+            const button = UI.widgets.button(dom, UI.icons.iconBase + 'noun_925021.svg', 'Drag here to delete')
+            async function droppedURIHandler (uri) {
```

Comment:
In my browser (Chrome) `uri` is an array of strings, which makes this function fail. Maybe do a test, e.g. `const uri2 = typeof uri === 'string' ? uri : uri[0]` ?